### PR TITLE
[DOCS] Completed the docs for the Navigation node.

### PR DESCRIPTION
**Pull request by NathanWarden**
I have another PR that renames navmesh_create to navmesh_add which is also renamed in this PR. https://github.com/godotengine/godot/pull/14676

So, it will likely not be good to merge this until that is accepted/rejected. Obviously if it's rejected I'll name it back to navmesh_create in this PR.

Thanks! :)

### CODE REVIEW

**Comment by NathanWarden** [comment]
Nevermind the navmesh_create/add rename. I was reminded that it could be added to the other PR and so I reverted the name here. :)

File: doc/classes/Navigation.xml [hunk 1]
```diff
@@ -55,6 +61,7 @@
 			<argument index="2" name="optimize" type="bool" default="true">
 			</argument>
 			<description>
+				Returns a path of points as a [code]PoolVector3Array[/code]. If the third optional [code]bool[/code] parameter is passed as false the [code]NavigationMesh[/code] agent properties will be taken into account, otherwise it will return the nearest path and ignore agent radius, height, etc.
```
Comment (Noshyaar) [inline doc/classes/Navigation.xml]: > If the third optional `bool` parameter is passed as false

IMO that's too long. Maybe: If `optimize` is false...

File: doc/classes/Navigation.xml [hunk 2]
```diff
@@ -85,11 +94,13 @@
 			<argument index="1" name="xform" type="Transform">
 			</argument>
 			<description>
+				Associates a [code]NavigationMesh[/code]'s id with a [code]Transform[/code]. Its position, rotation and scale are are based on the [code]Transform[/code] passed.
```
Comment (Noshyaar) [inline doc/classes/Navigation.xml]: > are are

File: doc/classes/Navigation.xml (hunk 1 above)
Comment (NathanWarden) [reply doc/classes/Navigation.xml]: Thanks, I agree :)

File: doc/classes/Navigation.xml (hunk 2 above)
Comment (NathanWarden) [reply doc/classes/Navigation.xml]: Are are matey, take a long walk down a short gang plank! :D

Comment (NathanWarden) [reply doc/classes/Navigation.xml]: Both should be good now @Noshyaar. Thanks for your feedback :)

**Pull request merged**